nsv13/code/modules/cargo/packs.dm

```diff
@@ -531,6 +531,52 @@
 					/obj/item/gun/energy/laser)
 	crate_name = "laser crate"
 
+/datum/supply_pack/security/mparmor
+	name = "Military Police Replacement Crate"
+	desc = "Three vests of MP armor vests complete with MP undershirts, everything to replace which fire razed to the ground."
+	cost = 3600
+	contains = list(/obj/item/clothing/suit/ship/squad/military_police,
+					/obj/item/clothing/suit/ship/squad/military_police,
+					/obj/item/clothing/suit/ship/squad/military_police,
+					/obj/item/clothing/under/ship/military_police,
+					/obj/item/clothing/under/ship/military_police,
+					/obj/item/clothing/under/ship/military_police)
+	crate_name = "MP armor crate"
+
+/datum/supply_pack/security/armory/m45single
+	name = "M1911 handgun Single-pack"
+	desc = "For the ones which know that .45 ACP is a superior cartridge"
```

Comment:
All of these crates should actually describe what is in them instead of being the same text copied three times

Reply:
Fixed with [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)